html_parser.js

```diff
@@ -147,21 +154,48 @@ function add_course_taken(json, line) {
 
 	// ap courses that do not count for credit do not have numbers / attributes for corresponding college courses
 	if(!contains(courseString, 'NO AP')) {
-		course.course.subject = line.substring(line.search('(FL|SP|S1|S2)') + 5, line.search('(FL|SP|S1|S2)') + 9).replace(' ', '');
+		course.subject = line.substring(line.search('(FL|SP|S1|S2)') + 5, line.search('(FL|SP|S1|S2)') + 9).replace(' ', '');
 		course.classId = courseString.substring(9, 14);
 	}
 
 	// locates the rest of the parameters with some regex magic
 	course.credithours = courseString.substring(18, 22);
 	course.season = new RegExp('(FL|SP|S1|S2)').exec(line)[0];
-	course.year = line.substring(line.search('(FL|SP|S1|S2)') + 2, line.search('(FL|SP|S1|S2)') + 4);
+    course.year = line.substring(line.search('(FL|SP|S1|S2)') + 2, line.search('(FL|SP|S1|S2)') + 4);
 
+    course.termId = get_termid(course.season, course.year);
 	// determines whether the course is 'in progress' or completed and sorts accordingly
 	if(contains(courseString, 'IP')) {
 		json.inprogress.classes.push(course);			
 	} else {
 		json.completed.classes.push(course);
-	}
+    }
+    console.log(course);
+}
+
+/**
+ * Determines the termId parameter of a course with the season and year of the course.
+ * @param {String} season   The season of the course ('FL', 'SP', 'S1', 'S2')
+ * @param {int} year        The year during which the course occurs (i.3. 2018, 2019)
+ * @return {int}            A six-digit integer representing the termId.
+ * @throws err              if the given year is not part of the enumeration specified.
+ */
+function get_termid(season, year) {
+    // Makes the assumption that all years will be in the 21st century
+    // As different technology will likely be used in 81 years, this is a valid assumption
+    let termid = "20" + year;
+    switch(season) {
+        case "FL":
+            return termid + "40";
+        case "SP":
+            return termid + "10";
+        case "S1":
+            return termid + "20";
+        case "S2":
+            return termid + "30";
+        default:
+            throw "The given season was not a member of the enumeration required."
```

Comment:
The numbers here are incorrect. Additionally, this switch-case is missing a case, summer full. See my earlier comment here for details: 

https://github.com/sandboxneu/scheduleneu/pull/9#issuecomment-504398461

Reply:
Some of the documentation can be found [here](https://github.com/ryanhugh/searchneu/blob/master/docs/API.md). 

Additionally I'm not sure whether the "year" of a class in the HTML audit is the same "year" as used in the SearchNEU representation (ex. 2019 starts in fall 2018, meaning fall 2018 semester's "year" is actually 2019).